### PR TITLE
Create Events Page: Split the toggle buttons into 2 sections, and make one of 3 toggles required 

### PR DESCRIPTION
**Pull request by JohnCox2211**
The Bonners toggle will automatically toggle for Bonner events, but can be changed by the user if they wish and other event types can also use this toggle. The toggle state is saved between create event and edit event. The toggles are divided as requested into required and optional categories as well.

### CODE REVIEW

**Comment by JohnCox2211** [comment]
https://github.com/BCStudentSoftwareDevTeam/celts/issues/1294 - github issue

https://trello.com/c/50TXOdHE - trello card

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
The bonner toggle only should be there for Bonner events.

Let's clean up the html classes and styles and try and simplify what controls the width of the components. Use the bootstrap grid only, if possible, for rows and columns.

File: app/static/js/createEvents.js [hunk 1]
```diff
@@ -98,10 +104,25 @@ $(document).ready(function () {
       $("#limitGroup").hide();
     }
   });
+  // Determine which checkbox was clicked and its current checked status, uncheck others
+  $("#checkIsTraining, #checkServiceHours, #checkBonners").on('click', function (event) {
+    let $target = $(event.target);
+    $("#checkIsTraining, #checkServiceHours, #checkBonners").not($target).prop('checked', false);
```
Comment (BrianRamsay) [inline app/static/js/createEvents.js]: we can just use `$(event.target)` in the `not()`

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Let's clean up the css so we are relying more on bootstrap. the cutoff point should be xl, then you don't need min-width or the @media selector

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**